lib/ui/profitloss/ProfitLossList.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:farmsmart_flutter/model/loading_status.dart';
+import 'package:farmsmart_flutter/redux/app/app_state.dart';
+import 'package:farmsmart_flutter/ui/profitloss/ProfitLossListItem.dart';
+import 'package:farmsmart_flutter/ui/profitloss/profit_loss_viewmodel.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
+
+class ProfitLossViewModel {
+  final String title;
+  final String titleDetail;
+  final String subTitle;
+
+  ProfitLossViewModel(this.title, this.titleDetail, this.subTitle);
+}
+
+ProfitLossViewModel buildProfitLossViewModel() {
+  return ProfitLossViewModel("2,150", "KSh", "▲ 498 (17.4%)");
+}
+
+abstract class ProfitLossStyle {
+
+  final TextStyle titleTextStyle;
+  final TextStyle titleDetailTextStyle;
+  final TextStyle subTitleTextStyle;
+
+  final Color floatingButtonBackgroundColor;
```

Comment:
try to avoid a name that hint at the implementation specifics. Maybe name this actionButtonBackgroundColour.
maybe if won't float in the future, but we´ll always have the button to perform the P and L action.